src/main/resources/templates/fragments/exclude.html

```diff
@@ -18,7 +19,7 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">Exclude ({{group
             </div>
         </div>
 
-        <div class="table-responsive-sm">
+        <span class="table-responsive-sm">
```

Comment:
This should stay as a div.

Reply:
Added another commit to branch resolving this issue.